microsoft-azure-sb/3.0/modules/ROOT/pages/azure-service-bus-connector-examples.adoc

```diff
@@ -1,10 +1,25 @@
 = Azure Service Bus Connector 3.0 Examples - Mule 4
- 
-// Add text to introduce the example section and say what one finds 
-// here and why it is helpful or necessary info.
 
-// Put Examples here -- You may need to obtain these from the 
-// code repo /demo folder.
+There are demos available that you can download and configure for your own environment to see different flows that use the Azure Service Bus Connector operations. 
```

Comment:
Try to avoid beginning sentences with "There is" or "There are." 
For some readers, there is a slight cognitive disconnect about the subject of the sentence, and it is a weird sort of passive voice.
```suggestion
To see different flows that use the Azure Service Bus Connector operations, import and configure for your own environment one of the available demo projects. 
```